ol-keycloak/oltheme/src/main/resources/theme/ol/login/login.ftl

```diff
@@ -1,7 +1,7 @@
 <#import "template.ftl" as layout>
 <@layout.registrationLayout displayMessage=!messagesPerField.existsError('username','password') displayInfo=realm.password && realm.registrationAllowed && !registrationDisabled??; section>
     <#if section = "header">
-        <#if attemptedName??>
+        <#if (attemptedName!"")?has_content>
```

Comment:
I had some changes in this branch earlier that did this as part of a larger refactor, but it was mainly related to the feature around a global direct link to Open, I'd probably prefer to either pull that in or deal with it independently, but either way I don't think it impacts what's needed in the template here. I made the change to additionally handle blank names because we don't want it showing up as "Hello, !".

Reply:
Here's the relevant code in my other branch: https://github.com/mitodl/ol-keycloak/compare/nl/open-link#diff-8940fd1470f886ebef38629b593030e62bd9a6b4efa6ffc182efd96e1d04ea82R40-R42